src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -95,17 +95,23 @@ const getResultIndicatorPeriods = () => {
 
   return {
     sixmonths: {
-      label: 'Siste 6 mnd',
+      label: i18n.t('period.sixmonths'),
       key: 'sixmonths',
       startDate: sixMonthsBack,
       endDate: currentDate,
     },
     year: {
-      label: 'Hittil i år',
+      label: i18n.t('period.year'),
       key: 'year',
       startDate: new Date(currentYear, 0, 1),
       endDate: currentDate,
     },
+    all: {
+      label: i18n.t('period.all'),
+      key: 'all',
+      startDate: false,
```

Comment:
Dette er kanskje bevisst, at den med `startDate: false` ikke også setter perioden i `DashboardPeriodSelector` (som de andre)? Det gir kanskje mening for dette valget, eller bør perioden bli satt fra og med første måling, som valget heter?